README.md

```diff
@@ -38,7 +38,7 @@ jobs:
     steps:
       - uses: actions/checkout@v4
```

Comment:
🔑 Ensure that the secrets used (OPENAI_API_KEY and GITHUB_TOKEN) are properly documented in the README for new developers. This enhances maintainability and helps avoid confusion.